services/backup_socrata.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+import datetime
+import os
+import requests
+
+import boto3
+
+from config.knack import CONFIG
+import utils
+
+BUCKET = os.getenv("BUCKET")
+AWS_ACCESS_ID = os.getenv("AWS_ACCESS_ID")
+AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
+SOCRATA_APP_TOKEN = os.getenv("SOCRATA_APP_TOKEN")
+
+
+def export_dataset(resource_id):
+    url = f"https://data.austintexas.gov/resource/{resource_id}.csv?$limit=99999999?$$app_token={SOCRATA_APP_TOKEN}"
+    res = requests.get(url)
+    data = res.text
```

Comment:
i went to see what would happen if you requested a nonexistent data set, if this would fail, but the res still has text
`'{\n  "code" : "dataset.missing",\n  "error" : true,\n  "message" : "Not found",\n  "data" : {\n    "id" : "hyfg-87hg"\n  }\n}\n'`

But maybe it should still alert if for some reason it wasnt able to get the dataset?

Reply:
yeah good point turns out my query to add the app token actually was wrong anyway so I added some error handling.